Resources/Views/Hub/_carousel.leaf

```diff
@@ -0,0 +1,81 @@
+
+                <div class="d-flex flex-row align-items-center mt-5 mb-3">
+                  <img src="#awsImage(session.ownerImageUrl)" class="d-block rounded" width="80" alt="#(session.ownerName)">
+                  <div class="ps-4">
+                    <h5 class="fw-medium fs-lg mb-1">#(session.title)</h5>
+                    <p class="fs-sm mb-0">#(session.ownerName)</p>
+                  </div>
+                </div>
+              
+              <div class="row flex-nowrap overflow-auto">
+              #for(slot in session.slots):
+                <div class="col-5 col-md-5 col-sm-5 col-lg-4 col-xl-3">
+                <div class="card #if(slot.isParticipant):bg-dark text-white#endif">
+                  <div class="card-body">
+                    <h5 class="card-title fs-sm" style="color: inherit;">#(slot.day) at #date(slot.date, "HH:mm")</h5>
+                    
+                    #if(session.maximumAttendance > 1):
+                    <p><span class="badge bg-info">#(session.maximumAttendance - slot.participantCount) spot left</span></p>
+                    #endif
+                    
+                    #if(session.maximumAttendance == 0):
+                    <a href="#" class="btn btn-sm btn-secondary disabled">Coming Soon</a>
+                    #elseif(!hasValidTicket):
+                    <a href="#" class="btn btn-sm btn-secondary disabled">Invalid</a>
+                    #elseif(slot.isParticipant):
+                    <a href="#" class="btn btn-sm btn-primary" data-bs-toggle="modal" data-bs-target="#slot-#(slot.id)">Cancel</a>
+                    #elseif(slot.isFullyBooked):
+                    <a href="#" class="btn btn-sm btn-secondary disabled">#if(session.maximumAttendance > 1):Fully Booked#else:Taken#endif</a>
```

Comment:
Can we change "Taken" for "Booked" in the multiple call sites? Feels like a "softer" word to use.